src/redux/books/books.js

```diff
@@ -0,0 +1,52 @@
+const ADD = 'book-store/books/ADD ';
+const REMOVE = 'book-store/books/REMOVE';
+
+const initialState = {
+  books: [
+    {
+      id: '1',
+      author: 'Gayle Laakmann McDowell',
+      title: 'Cracking the Coding Interview',
+    },
+    {
+      id: '2',
+      author: 'Robert Greene',
+      title: 'The 48 Laws of Power',
+    },
+  ],
+};
+
+export function add(book) {
+  return {
+    type: ADD,
+    obj: book,
+  };
+}
+
+export function remove(id) {
+  return {
+    type: REMOVE,
+    obj: id,
+  };
+}
+
+export default function reducer(state = initialState, action) {
```

Comment:
- [ ] It would be better if you use arrow function instead of the keyword `function`